lerobot/common/robot_devices/cameras/intelrealsense.py

```diff
@@ -28,22 +29,23 @@
 SERIAL_NUMBER_INDEX = 1
 
 
-def find_camera_indices(raise_when_empty=True) -> list[int]:
+def find_cameras_info(raise_when_empty=True) -> dict[int, str]:
     """
-    Find the serial numbers of the Intel RealSense cameras
+    Find the names and the serial numbers of the Intel RealSense cameras
     connected to the computer.
     """
-    camera_ids = []
+    cameras_info = {}
     for device in rs.context().query_devices():
         serial_number = int(device.get_info(rs.camera_info(SERIAL_NUMBER_INDEX)))
-        camera_ids.append(serial_number)
+        name = device.get_info(rs.camera_info.name)
+        cameras_info[serial_number] = name
 
-    if raise_when_empty and len(camera_ids) == 0:
+    if raise_when_empty and len(cameras_info) == 0:
         raise OSError(
             "Not a single camera was detected. Try re-plugging, or re-installing `librealsense` and its python wrapper `pyrealsense2`, or updating the firmware."
         )
 
-    return camera_ids
+    return cameras_info
```

Comment:
I am not fan of modifying this function, to stay consistent with the one in opencv.py 
Instead, could we have another function that given camera_indices / serial number retrieves the camera name?

Reply:
What can we do to unify opencv and intelrealsense on this function?

I would imagine `find_cameras_info(raise_when_empty=True) -> list[dict]:` for both, with `camera_infos[0]["index"]` being the camera index port for opencv or intelrealsense serial, and `camera_infos[0]["name"]` not existing for opencv.

Reply:
Nit: `camera_infos` instead of `cameras_info` for consistency with `camera_ids` and how we usually add a `s` at the end for our variable names of type `list`

Reply:
> What can we do to unify opencv and intelrealsense on this function?

As we discussed, I think we should eventually converge to a single `Camera` class with different backends (realsense, opencv...). This would probably simplify the design and avoid having to monkeypatch those classes in the tests. Then we could think about the design of this function (which there would be a single one). Wdyt?

> Nit: camera_infos instead of cameras_info for consistency with camera_ids and how we usually add a s at the end for our variable names of type list

I wondered about the exact same thing but the plural of "info" in English is without a s.

Reply:
1. I agree, but I think we should homogenise opencv and intelrealsense at this level before merging. What do you think of my suggestion? 

2. I know but we dont write in english, we write in python (which is an english dialect) so the english grammar rules dont apply the same way ^^